### ISSUE TITLE
Harad Tribalism

### ISSUE THREAD

**Issue by LeGrandiSlayer** [Lonhaldar/MiddleEarth-Project#1]
In the current version of the mod, All Haradrim and Barangils provinces are tribal, with no castles and cities for Near-Harad. While it looked cool when it was discussed, I do not think it was good for balance in the area, and for Gondor, because now Harad is laughably weak, and expansion by the BN into Harad is not encouraged, because a lot of the provinces are single holding tribes, which give less levies than castles un-upgraded.

Therefor, I believe that Harad should return to the light of Feudalism, and it's tribal holdings done away with in the name of balance, because we can find other ways to make Harad fun and different to play as, or against.

**Comment by Lonhaldar** [comment]
I agree with you. Even if tribes are cool, it makes the game unbalanced imho.

**Comment by Porkenstein** [comment]
What about a middle ground?  Khand and Far Haradrim could be tribal vassals
of feudal kings.

**Comment by LeGrandiSlayer** [comment]
I don't think even Far Harad should have tribal, that should be reserved either for all low(primitive) men or, to be frank, not in at all. As for Khand, that depends on if we make the Easterlings nomadic or not(personally, I don't think we should, thanks to how hard coded they are and how much weirdness they produce).

**Comment by ghost** [comment]
I'm with you guys, the tribalism doesn't really help the situation of the Haradrim. I'd also be in favour of scrapping the Nomad governments altogether. Too much strange stuff is happening (as GrandiSlayer said) with the realms quickly becoming a mess of feudal counties mixed against a weak Nomadic empire.

**Comment by Lonhaldar** [comment]
So let's do it! I'm agree with that, Nomadics governments broke the balance, and created some weirdness sometimes.

**Comment by LeGrandiSlayer** [comment]
The last of the tribal governments have been removed, I'll do a quick check to see if Rhun still has some nomads hanging around.

**Comment by Lonhaldar** [comment]
Great!
